Projects/Feature/Sources/App/AppFeature.swift

```diff
@@ -0,0 +1,82 @@
+//
+//  AppFeature.swift
+//  Feature
+//
+//  Created by JongHoon on 7/23/24.
+//
+
+import Foundation
+
+import FeatureLoginInterface
+import FeatureLogin
+
+import ComposableArchitecture
+
+@Reducer
+public struct AppFeature {
+  @ObservableState
+  public struct State: Equatable {
+    public var appDelegate: AppDelegateFeature.State
+    var tabView: MainTabViewFeature.State?
+    var login: LoginFeature.State?
+    
+    public init() {
+      self.appDelegate = .init()
+    }
+  }
+  
+  public enum Action {
+    case onAppear
+    case loginCheckComplted(isLoggedIn: Bool)
+    case appDelegate(AppDelegateFeature.Action)
+    case tabView(MainTabViewFeature.Action)
+    case login(LoginFeature.Action)
```

Comment:
다른 Feature의 Action이랑 State는 뒤에 표시해두자!

```suggestion
    case appDelegateAction(AppDelegateFeature.Action)
    case tabViewAction(MainTabViewFeature.Action)
    case loginAction(LoginFeature.Action)
```

Reply:
이거 수정할게!